### PR TITLE
MU blueprints: enable `EMBER_MODULE_UNIFICATION` feature flag

### PR DESCRIPTION
**Pull request by ppcano**
Solves the same issue than https://github.com/ember-cli/ember-cli/pull/8433

cc @NullVoxPopuli

### CODE REVIEW

File: packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
```
Comment (NullVoxPopuli) [inline packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js]: the comment should probably be updated as well

Comment (ppcano) [reply packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js]: https://github.com/ember-cli/ember-octane-blueprint/pull/35/commits/55005de0d26d9932170a47ab7b5ae42397e11133

Comment (ppcano) [reply packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js]: @NullVoxPopuli Done

**Comment by NullVoxPopuli** [comment]
is this change required from a recent version of ember-source?

in my https://emberclear.io  app, changing from 
```js
        'ember-module-unification': true,
```
to screaming snake case causes a bunch of my tests to fail.

**Comment by ppcano** [comment]
I guess so, the change is needed for https://github.com/ember-cli/ember-cli/pull/8431 and https://github.com/ember-cli/ember-cli/issues/8411

I think the MU feature was never enabled in Ember. 

What are the errors?

**Comment by NullVoxPopuli** [comment]
it's a bunch of element not found errors in my acceptance tests.
clicking through the app though, no errors appear in the console. so.. :man_shrugging:

**Comment by ppcano** [comment]
Regarding the error in your MU app, I think it may be good if you test this change and report any issue.

MU users have likely used the `module-unification` or `octane` blueprints where they have enabled the `EMBER_CLI_MODULE_UNIFICATION` flag but not the `EMBER_MODULE_UNIFICATION`flag in Ember which does a few changes.

The MU changes in Ember are mainly related to the addon consumption (using the addon namespace) what may not affect most of the existing MU apps. 

Anyway, it may be good to put the 👀on it.

**Pull request merged**